src/middleware/validate/util.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright 2017-2022 Parity Technologies (UK) Ltd.
+// This file is part of Substrate API Sidecar.
+//
+// Substrate API Sidecar is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import { Request, Response } from 'express';
+import { RequestHandler } from 'express-serve-static-core';
+
+/**
+ * Assert that a middleware does not error with a the given `req`.
+ *
+ * @param name thing it does not error on
```

Comment:
I don't understand the comment `name thing`, it looks like you just moved the code but would be good with some docs that I could understand :)

Reply:
`@param name` is the name of the param, and then `thing it does not error on` is the comment, IIRC. It is confusing to see it all together like this though :)

Reply:
Yea with inline styling it is way more clear, but like @jsdw it is just the parameter it is mentioning. I also cleaned up the docs here to be more clear.